content/push/configure/web.textile

```diff
@@ -83,12 +83,41 @@ The following diagram demonstrates the direct activation process:
   <img src="@content/diagrams/push-direct-activation-web-push.png" style="width: 100%" alt="push notifications in Ably">
 </a>
 
+h4(#package-man). Activate via package manager
+
+If you have installed the Ably SDK using a package manager (such as npm), you can use the @activate@ method to enable push notifications. You can then initialize the push plugin with the Ably client instance.
+
 The following example initializes an instance of the Ably realtime service and then activates the push notifications feature _for that instance_:
 
 ```[javascript]
-await ably.push.activate();
+await client.push.activate();
 ```
 
+h4(#no-package-man). Activate via Content Delivery Network (CDN)
+
+If you can't use a package manager, you can activate it via CDN by directly loading the push plugin in your HTML using a @<script>@ tag.
+
+The following example is loading the Ably push notifications library from a CDN:
+
+```[html]
+<script src="https://cdn.ably.com/lib/push.umd.min-2.js"></script>
+```
+
+When loaded, the push plugin becomes available on the global object via the @AblyPushPlugin@ property. You can then initialize the push plugin with the Ably client instance.
+
+The following example initializes an instance of the Ably realtime service and then activates the push notifications feature _for that instance_:
```

Comment:
I'm not sure this line is entirely accurate as the example shows instantiating a REST client. 

I think generally most browser clients are going to be using realtime however so I'm not sure why the examples in the GitHub repo show using REST. Probably worth checking with the ecosystems team

Reply:
it will work both for Rest and Realtime clients. we can change the example here to be:
```
const client = new Ably.Realtime({
  ...options,
  plugins: { Push: AblyPushPlugin },
});
```
so that the language above matches the example

Reply:
Thanks for the feedback guys! @mclark-ably please see fixup 9357e3a10697242dfafccd3a049521c64f97d8e7